test/parallel/test-fs-symlink-dir-junction-relative.js

```diff
@@ -23,14 +23,15 @@
 // Test creating and resolving relative junction or symbolic link
 
 const common = require('../common');
+const fixtures = require('../common/fixtures');
 const assert = require('assert');
 const path = require('path');
 const fs = require('fs');
 
 const linkPath1 = path.join(common.tmpDir, 'junction1');
 const linkPath2 = path.join(common.tmpDir, 'junction2');
-const linkTarget = path.join(common.fixturesDir);
-const linkData = path.join(common.fixturesDir);
+const linkTarget = path.join(fixtures.fixturesDir);
```

Comment:
I bet this could just be changed to `const linkTarget = fixtures.fixuresDir;`

Reply:
@rmg maybe this could be handled when it lands? See https://github.com/nodejs/code-and-learn/issues/70#issuecomment-334893709. I know this isn't really a nit such as message line lengths, but I agree with @gibfahn that force pushes and amended commits are a bit of a high barrier for `code-and-learn` first time contributors.

Reply:
Force pushing isn't required for this, @genewoo if you would be willing to make the change @rmg suggested that would be great, you can change the line and push another commit to your branch.

Agreed that this isn't a blocker (which I assume is why @rmg didn't request changes).